spec/socialprovider.spec.js

```diff
@@ -102,6 +102,19 @@ describe("Tests for message batching in Social provider", function() {
     expect(xmppSocialProvider.client.send).toHaveBeenCalled();
   });
 
+  it("sends message to correct destinations", function() {
+    xmppSocialProvider.sendMessage('Bob', 'Hi', function() {});
+    xmppSocialProvider.sendMessage('Alice', 'Hi', function() {});
+    expect(xmppSocialProvider.client.send).not.toHaveBeenCalled();
+    jasmine.clock().tick(100);
+    expect(xmppSocialProvider.client.send.calls.count()).toEqual(2);
+    var dest = xmppSocialProvider.client.send.calls.first().args[0].parent.attrs.to;
+    if (dest !== 'Bob' && dest !== 'Alice') {
+      expect(dest).toEqual('Bob');
```

Comment:
Yeah - this is expressing that it either needs to be 'bob' or 'alice', since the messages can be reordered depending on how object keys are implemented.  There isn't a direct jasmine way to say .toEqual('bob' or 'alice'), so If it isn't one of the two I want it to fail.  better way to express that?